README.md

```diff
@@ -1,12 +1,12 @@
 <a href="https://www.sktime.net"><img src="https://github.com/sktime/sktime/blob/main/docs/source/images/sktime-logo.svg" width="175" align="right" /></a>
 
-# Welcome to sktime
+# Welcome to sktime :hugs:
```

Comment:
I think this is too close to the "huggingface" logo.

How about 🌞 ? Or 👋 , 🚀 , 🌠 , 🌅 ?

Reply:
yes, it is the shortcode for "huggingface".
I got a better one on your suggestions. (💫) (:dizzy:)